### PR TITLE
fix photocolor

### PR DESCRIPTION
**Pull request by mixcas**
run analysis after photo taken is loaded

### CODE REVIEW

File: cordova/www/games/photocolor/photocolor.js [hunk 1]
```diff
@@ -101,44 +101,45 @@ var Photocolor = {
...
+          result = true;
+          arrayMatch = i;
+          break;
+        }
+      }
 
-    var result = false;
+      if (result) {
 
-    for (var i = 0; i < paletteArray.length; i++) {
```
Comment (mixcas) [inline cordova/www/games/photocolor/photocolor.js]: this was runnning (maybe cuz async) before photo was loaded

**Comment by varffs** [comment]
I cant pull cus tho this fixes the analysis there is an issue with the clearing image function not working properly on failed analysis attempts. Perhaps we never show the photo taken to the end user?

**Comment by mixcas** [comment]
@varffs ill check it

**Comment by mixcas** [comment]
@varffs i can't reproduce the bug u r talking about
i debugged the code for that function and runs fine

**Pull request merged**